templates/core/index.html

```diff
@@ -588,21 +642,23 @@ <h4>Manufacturer update control</h4>
   <div class="row u-vertically-center u-equal-height">
     <div class="col-6 u-hide--small u-align--center u-vertically-center">
       {{
-        image(
-          url="https://assets.ubuntu.com/v1/bd695c08-app+store.svg",
-          alt="",
-          height="200",
-          width="200",
-          hi_def=True,
-          loading="lazy"
-        ) | safe
+      image(
+      url="https://assets.ubuntu.com/v1/bd695c08-app+store.svg",
+      alt="",
+      height="200",
+      width="200",
+      hi_def=True,
+      loading="lazy"
+      ) | safe
       }}
     </div>
     <div class="col-6">
       <h4>App store included</h4>
       <p class='p-heading--five'>Your own app store and software curation</p>
-      <p>Every device built with Ubuntu Core has a secure app store. Use the global store for access to all the standard apps, or curate your own collection.</p>
-      <p><a class="p-button--neutral is-inline" href="/internet-of-things/appstore" style="margin-left: 0;"><span class="p-link--external">See what an app store can do for your things</span></a></p>
+      <p>Every device built with Ubuntu Core has a secure app store. Use the global store for access to all the standard
+        apps, or curate your own collection.</p>
+      <p><a class="p-button--neutral is-inline" href="/internet-of-things/appstore" style="margin-left: 0;"><span
+            class="p-link--external">See what an app store can do for your things</span></a></p>
```

Comment:
extra newline, also, your adding a lot of whitespace changes, perhaps your editor is doing this?